### ISSUE TITLE
Document partial bam download in Ag3 user guide

### ISSUE THREAD

**Issue by alimanfoo** [malariagen/vector-data#23]
Document how to use bai files to download regions from bam files, as [suggested here](https://github.com/malariagen/vector-data/discussions/19#discussioncomment-594918).

**Comment by jonbrenas** [comment]
I don't think we actually want to do that. @cclarkson, @ahernank, what do you think?
